apps/vmq_server/src/vmq_info.erl

```diff
@@ -0,0 +1,169 @@
+%% Copyright 2016 Erlio GmbH Basel Switzerland (http://erl.io)
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(vmq_info).
+-behaviour(vmq_ql_query).
+-include("vmq_server.hrl").
+-include_lib("vmq_ql/include/vmq_ql.hrl").
+
+-export([fields_config/0,
+         fold_init_rows/2]).
+
+%% used by vmq_info_cli
+-export([session_info_items/0]).
+
+fields_config() ->
+    QueueBase = #vmq_ql_table{
+                   name =       queue_base,
+                   depends_on = [],
+                   provides =   [node, mountpoint, client_id, queue_pid],
+                   init_fun =   fun row_init/1,
+                   include_if_all = true
+                  },
+    Queues = #vmq_ql_table{
+                   name =       queues,
+                   depends_on = [QueueBase],
+                   provides = [status,
```

Comment:
AFAICT the `status` will never be part of the map returned from `vmq_queue:info/1` and will therefore always default to `null`.